cli/cmd/plugin/ui/web/tanzu-ui/src/shared/components/ClusterInfo/ClusterInfo.tsx

```diff
@@ -0,0 +1,64 @@
+// React imports
+import React from 'react';
+
+// Library imports
+import { CdsButton } from '@cds/react/button';
+import { CdsIcon } from '@cds/react/icon';
+import { CdsDivider } from '@cds/react/divider';
+
+import './ClusterInfo.scss';
+
+function ClusterInfo(props: any) {
```

Comment:
Since this is not a shared component across management, workload and unmanaged clusters I would rename it to something like UnmanagedClusterCard